aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -458,6 +459,12 @@ Resources:
 <% if database-%>
 <%=  component 'database'%>
 <% end -%>
+  ChefConfig:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Description: !Sub "Custom Chef attributes for ${AWS::StackName} CloudFormation stack"
+      Name: !Sub "CfnStack/${AWS::StackName}/chef"
+      SecretString: '{}'
```

Comment:
Might be useful to mention that we manage the values stored in this Secret manually (outside of this CloudFormation template).

Reply:
done in 4155489da9126653d1ebe4457f948e649e457e40.